src/workerd/io/limit-enforcer.h

```diff
@@ -12,119 +12,116 @@ namespace workerd {
   // filled in to indicate what happened, otherwise it is left null.
+  virtual kj::Own<void> enterStartupJs(
+      jsg::Lock& lock, kj::Maybe<kj::Exception>& error) const = 0;
 
+  // Like enterStartupJs(), but used when compiling a dynamically-imported module.
   virtual kj::Own<void> enterDynamicImportJs(
       jsg::Lock& lock, kj::Maybe<kj::Exception>& error) const = 0;
-  // Like enterStartupJs(), but used when compiling a dynamically-imported module.
 
-  virtual kj::Own<void> enterLoggingJs(
-      jsg::Lock& lock, kj::Maybe<kj::Exception>& error) const = 0;
   // Like enterStartupJs(), but used to enforce tight limits in cases where we just intend
   // to log an error to the inspector or the like.
+  virtual kj::Own<void> enterLoggingJs(
+      jsg::Lock& lock, kj::Maybe<kj::Exception>& error) const = 0;
 
+  // Like enterStartupJs(), but used when receiving commands via the inspector protocol.
   virtual kj::Own<void> enterInspectorJs(
       jsg::Lock& lock, kj::Maybe<kj::Exception>& error) const = 0;
-  // Like enterStartupJs(), but used when receiving commands via the inspector protocol.
 
-  virtual void completedRequest(kj::StringPtr id) const = 0;
   // Notifies the enforcer that a request has been completed. The enforcer is more lenient about
   // limits if several requests have been completed, vs. if limits are broken right off the bat.
+  virtual void completedRequest(kj::StringPtr id) const = 0;
 
-  virtual bool exitJs(jsg::Lock& lock) const = 0;
   // Called whenever exiting JavaScript execution (i.e. releasing the isolate lock). The enforcer
   // may perform some resource usage checks at this time.
   //
   // Returns true if the isolate has exceeded limits and become condemned.
+  virtual bool exitJs(jsg::Lock& lock) const = 0;
 
-  virtual void reportMetrics(IsolateObserver& isolateMetrics) const = 0;
   // Report resource usage metrics to the given isolate metrics object.
+  virtual void reportMetrics(IsolateObserver& isolateMetrics) const = 0;
 };
 
+// Abstract interface that enforces resource limits on a IoContext.
 class LimitEnforcer  {
-  // Abstract interface that enforces resource limits on a IoContext.
-
 public:
-  virtual kj::Own<void> enterJs(jsg::Lock& lock, IoContext& context) = 0;
   // Called just after taking the isolate lock, before executing JavaScript code, to enforce
   // limits on that code execution, particularly the CPU limit. The returned `Own<void>` should
   // be dropped when JavaScript is done, before unlocking the isolate.
+  virtual kj::Own<void> enterJs(jsg::Lock& lock, IoContext& context) = 0;
 
-  virtual void topUpActor() = 0;
   // Called on each new event delivered that should cause an actor's resource limits to be
   // "topped up". This method does nothing if the IoContext is not an actor. Note that this must
   // not be called while in a JS scope, i.e. when `enterJs()` has been called and the returned
   // object not yet dropped.
-  //
+  virtual void topUpActor() = 0;
```

Comment:
```suggestion
  virtual void topUpActor() = 0;

```

Reply:
I intentionally left the TODO right up next to the `topUpActor()` without a space as they go together.